### PR TITLE
add --wait-for-output | --wfo  and --wait-for-active | --wfa option to zos-jobs submit commands 

### PR DESCRIPTION
**Pull request by ChrisBoehmCA**
* Try with `zowe zos-jobs submit data-set my.data.set --wait-for-output` and  `zowe zos-jobs submit data-set my.data.set --wait-for-active`
* if --vasc is not specified, the command will return the IJob object so that the return code can be easily checked without issuing `zowe zos-jobs view job-status-by-jobid <jobid>`

### CODE REVIEW

File: packages/zosfiles/__tests__/cli/create/ussDir/__snapshots__/ussDir.definition.unit.test.ts.snap [hunk 1]
```diff
@@ -29,33 +29,3 @@ Array [
   },
 ]
 `;
-
```
Comment (tucker01) [inline packages/zosfiles/__tests__/cli/create/ussDir/__snapshots__/ussDir.definition.unit.test.ts.snap]: Were these obsolete?

File: packages/zosjobs/src/cli/submit/stdin/stdin.definition.ts [hunk 2]
```diff
@@ -27,6 +27,17 @@ export const StdinDefinition: ICommandDefinition = {
                 " If you use this option you will wait the job to complete.",
             type: "boolean"
         },
+        {
+            name: "wait-for-output", aliases: ["wfo"],
```
Comment (tucker01) [inline packages/zosjobs/src/cli/submit/stdin/stdin.definition.ts]: Should this have a `conflictsWith` property as well? (for `vasc` and `directory`)?

Comment (tucker01) [reply packages/zosjobs/src/cli/submit/stdin/stdin.definition.ts]: Same for the other definitions on the other submit commands.

File: packages/zosjobs/__tests__/__system__/cli/submit/stdin/__scripts__/submit_valid_stdin_wait.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+# pass the JCL as stdin to this script
+cat $1 | zowe zos-jobs submit stdin -w
```
Comment (tucker01) [inline packages/zosjobs/__tests__/__system__/cli/submit/stdin/__scripts__/submit_valid_stdin_wait.sh]: This option doesn't look right? I thought the aliases were `wfo` and `wfa`?

File: packages/zosjobs/__tests__/__system__/cli/submit/local-file/__scripts__/submit_valid_local_file_wait.sh [hunk 4]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+# pass the data set name as an argument to the script
+zowe zos-jobs submit local-file $1 --wait
```
Comment (tucker01) [inline packages/zosjobs/__tests__/__system__/cli/submit/local-file/__scripts__/submit_valid_local_file_wait.sh]: Aren't the aliases `wfo` and `wfa`?

File: packages/zosjobs/__tests__/__system__/cli/submit/data-set/__scripts__/submit_valid_data_set_wait.sh [hunk 5]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+# pass the data set name as an argument to the script
+zowe zos-jobs submit data-set $1 -w
```
Comment (tucker01) [inline packages/zosjobs/__tests__/__system__/cli/submit/data-set/__scripts__/submit_valid_data_set_wait.sh]: This doesn't look right? Aren't the aliases `wfo` and `wfa`?

**Comment by ChrisBoehmCA** [comment]
Thanks, sorry I left the -w in the scripts I will fix that.  And the conflictsWith and integration tests.

File: packages/zosfiles/__tests__/cli/create/ussDir/__snapshots__/ussDir.definition.unit.test.ts.snap (hunk 1 above)
Comment (ChrisBoehmCA) [reply packages/zosfiles/__tests__/cli/create/ussDir/__snapshots__/ussDir.definition.unit.test.ts.snap]: Yeah i did -u  on the unit test script so it must have auto removed this

File: packages/zosjobs/src/cli/submit/stdin/stdin.definition.ts (hunk 2 above)
Comment (ChrisBoehmCA) [reply packages/zosjobs/src/cli/submit/stdin/stdin.definition.ts]: I'm not sure actually. --directory and --vasc both involve waiting for output so in a way they don't conflict, right? Or am I misunderstanding?

Comment (tucker01) [reply packages/zosjobs/src/cli/submit/stdin/stdin.definition.ts]: Yeah, sorry not wait for output, but wait for active?

Comment (ChrisBoehmCA) [reply packages/zosjobs/src/cli/submit/stdin/stdin.definition.ts]: Oh, yes you're right

**Comment by tucker01** [comment]
Signed-off-by: tucker01 <jason.tucker@broadcom.com>

**Pull request merged**